src/address.ts

```diff
@@ -16,6 +16,7 @@ export class Address {
                public readonly postcode: string,
                public readonly postcodeType: string,
                public readonly formattedAddress: string,
-               public readonly point: Point) {
+               public readonly point: Point,
+               public readonly udprn?: string){
```

Comment:
is this always sent back? I would assume it is so I would signify it would always be present in the api with:
```suggestion
               public readonly udprn: string){
```

Reply:
or
```suggestion
               public readonly udprn: string | undefined){
```
as per some of the other fields